kompose.spec

```diff
@@ -1001,6 +1033,9 @@ export LDFLAGS=%{ldflags}
 %endif
 
 %changelog
+* Fri Jul 21 2017 Suraj Deshmukh <surajssd009005@gmail.com> - 0-0.1.git253ec14
```

Comment:
Not sure about this, this is auto generated by `gofed`, also older entries in changelog are like that.

Do you suggest me to change this and all other entries?